packages/neo-one-server-plugin-project/src/__e2e__/buildCommandJs.test.ts

```diff
@@ -0,0 +1,459 @@
+import {
```

Comment:
Can you extract out the common logic between this file and `buildCommand.test.ts`

Reply:
There's lots of "any" in this solution: https://github.com/neo-one-suite/neo-one/pull/785/files#diff-28076ab6b09c780f81fa81b6b75c4aaaR11

Reply:
👍